src/sql/workbench/services/objectExplorer/browser/treeCreationUtils.ts

```diff
@@ -143,5 +142,5 @@ export class TreeCreationUtils {
 }
 
 function useAsyncServerTree(configurationService: IConfigurationService): boolean {
-	return configurationService.getValue<boolean>(CONFIG_WORKBENCH_ENABLEPREVIEWFEATURES) && configurationService.getValue<boolean>('serverTree.useAsyncServerTree');
+	return configurationService.getValue<boolean>('serverTree.useAsyncServerTree');
```

Comment:
If someone has preview features disabled it feels like we should still be ignoring this setting.

Reply:
The original idea is to make this the main tree in the next release. I worry if we hide it behind the preview flag, we might reduce the number of people testing it in insiders.

Reply:
That plan seems fine to me (I assume you'll be making sure we have plenty of people testing it) - but if you're going that route I'd remove the (Preview) text from the setting description then. I just don't think we should have any "preview" functionality that isn't dependent on the preview features flag as well - but in this case it seems like less of preview functionality and more like just a general improvement to the core feature.